pythosf/client.py

```diff
@@ -27,10 +28,12 @@ def base_headers(token):
         }
 
     def json_api_request(self, url, method=None, item_id=None, item_type=None, attributes=None, raw_body=None,
-                         query_parameters=None, fields=None, headers=None, retry=True, token=None):
+                         query_parameters=None, fields=None, headers=None, retry=True, token=None, auth=None):
```

Comment:
Is there an auth type that works with our token? If not, we should make one. Maybe I'll deprecate out the old token format once we're sure that works well and I can clean up a bunch of code.

Reply:
Will check!